tests/test_spec.py

```diff
@@ -334,10 +338,55 @@ def test_calibrate_dispersion_model():
     print(f"Worst case wavelength disagreement from the test input model: {worst_case_wavlen_error:.2f} nm")
     assert worst_case_wavlen_error == pytest.approx(0, abs=0.5)
     print("Dispersion profile fit test passed.")
+
+def test_wave_cal():
+    """
+    test l3_to_l4.wave_cal(), the generation of the wavelength map extensions
```

Comment:
should comment in the docstring that this test requires running `test_calibrate_dispersion_model()` beforehand.